src/test/java/HW20.java

```diff
@@ -0,0 +1,85 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HW20 extends BaseTest {
+
+    @Test
+    public void DeletePlaylist() throws InterruptedException {
+        login("zahinedu@gmail.com", "te$t$tudent");
+        clickNewPlaylist();
+        createPlayList();
+        deletePlaylist();
+        playListDeleted();
+
+    }
+
+
+    public void createPlayList() {
+
+
+        WebElement newPlayList = driver.findElement(By.cssSelector(".fa-plus-circle"));
+        wait.until(ExpectedConditions.visibilityOf(newPlayList));
+        newPlayList.click();
+
+
+        WebElement addNewPlayList = driver.findElement(By.cssSelector("[data-testid='playlist-context-menu-create-simple']"));
+        wait.until(ExpectedConditions.visibilityOf(addNewPlayList));
+        addNewPlayList.click();
+
+
+        WebElement namePlayList = driver.findElement(By.xpath("//input[@name='name']"));
+        wait.until(ExpectedConditions.visibilityOf(namePlayList));
+        namePlayList.click();
+        namePlayList.clear();
+        namePlayList.sendKeys("Amira's List", Keys.ENTER);
+
+
+    }
+
+    public void deletePlaylist() {
+        WebElement deleteList = driver.findElement(By.xpath("//button[@title='Delete this playlist']"));
+        wait.until(ExpectedConditions.visibilityOf(deleteList));
+        deleteList.click();
+    }
+
+
+    public void playListDeleted() {
```

Comment:
Please, remove code redundancies or rename this method

![image](https://user-images.githubusercontent.com/104761482/212020529-8c28ff94-3a0c-41d5-a613-7b8bf040dd4d.png)


For example `public void enterButtonDeletePlayList()` 

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )